app/templates/components/table-workflow.hbs

```diff
@@ -44,7 +44,9 @@
   </div>
 </li>
 {{#each pagedContent as |workflow|}}
-  {{workflows.table-item workflow=workflow}}
+  {{#unless workflow.isDeleted}}
```

Comment:
O workflow é deletado da base, mas continua como um objeto no array, apontando pra nada. Uma maneira era fazer um filterBy isDeletedFalse, que no fundo faria o reload do model sempre. Ou fazer um bind no confirm do usuario, pra fazer um removeObject, pq os componentes são diferentes. Pensamos e esse parecia o jeito mais prático e barato de se fazer, quando for recarregado, esse unless não irá retirar nenhum elemento, mas quando um for retirado por algum delete, ele atualizará instantaneamente o template.